modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/extra/remote-app/src/routes/SalesRequestForm/steps/Review/index.tsx

```diff
@@ -0,0 +1,202 @@
+import DealRegistrationStepProps from '../../interfaces/dealRegistrationStepProps';
+
+const Review = ({
+	onCancel,
+	onSaveAsDraft,
+}: PRMFormikPageProps & DealRegistrationStepProps<DealRegistration>) => {
+	const {values, ...formikHelpers} = useFormikContext<DealRegistration>();
+
+	return (
+		<>
+			<PRMForm
+				className="bg-neutral-0"
+				description="Lorem ipsum dolor sit amet, consectetur adipiscing elit ut aliquam, purus sit amet luctus"
+				name="review"
+				title="Review Deal Registration"
+			/>
+
+			<PRMForm name="general" title="Deal Registration">
+				<Table
+					items={[
+						{
+							title: 'Partner Account Name',
+							value: 'Deathray Parent-A*',
+						},
+						{
+							title: 'MDF Activity Associated',
+							value: 'US',
+						},
+					]}
+					title="General Details"
+				/>
+
+				<Table
+					items={[
+						{
+							title: 'Account Name',
+							value: '{}',
+						},
+						{
+							title: 'Industry',
+							value: 'Insurance',
+						},
+						{
+							title: 'Address',
+							value: '...',
+						},
+						{
+							title: 'City',
+							value: 'Los Angeles',
+						},
+						{
+							title: 'Postal Code',
+							value: '91765',
+						},
+						{
+							title: 'State',
+							value: 'California',
+						},
+						{
+							title: 'Country',
+							value: 'US',
+						},
+					]}
+					title="Prospect Information"
+				/>
+
+				<Table
+					items={[
+						{
+							title: 'First Name',
+							value: 'Jane',
+						},
+						{
+							title: 'Last Name',
+							value: 'Doe',
+						},
+						{
+							title: 'Email Address',
+							value: 'jane@example.com',
+						},
+						{
+							title: 'Phone',
+							value: '(714) 718-4565',
+						},
+						{
+							title: 'Department',
+							value: 'Business - Marketing',
+						},
+						{
+							title: 'Job Role',
+							value: 'Project Manager',
+						},
+					]}
+					title="Primary Prospect Contact"
+				/>
+
+				<Table
+					items={[
+						{
+							title: 'First Name',
+							value: 'Jane',
+						},
+						{
+							title: 'Last Name',
+							value: 'Doe',
+						},
+						{
+							title: 'Email Address',
+							value: 'jane@example.com',
+						},
+					]}
+					title="Additional Contacts"
+				/>
+
+				<Table
+					items={[
+						{
+							title: 'Additional Information',
+							value: '...',
+						},
+					]}
+					title="Deal Information"
+				/>
+
+				<Table
+					items={[
+						{
+							title: 'Project Need',
+							value: '...',
+						},
+						{
+							title: 'Project Solution Categories',
+							value: '...',
+						},
+					]}
+					title="Project Information"
+				/>
+
+				<Table
+					items={[
+						{
+							title: 'Project Timeline',
+							value: '...',
+						},
+						{
+							title: 'Project Solution Categories',
+							value: '...',
+						},
+					]}
+					title="Business Objectives"
+				/>
+			</PRMForm>
+
+			<div className="bg-neutral-0">
+				<PRMForm.Footer>
```

Comment:
```suggestion
				<PRMForm.Footer className="bg-neutral-0">
```

Eh necessario editar o PRMForm.Footer para aceitar a prop className